x/slashing/keeper_test.go

```diff
@@ -215,13 +242,13 @@ func TestHandleAbsentValidator(t *testing.T) {
 	// validator should have been slashed
 	pool = sk.GetPool(ctx)
 	slashAmt := sdk.NewDec(amtInt).Mul(keeper.SlashFractionDowntime(ctx)).RoundInt64()
-	require.Equal(t, amtInt-slashAmt, pool.BondedTokens.RoundInt64())
+	require.Equal(t, amtInt-slashAmt-5, pool.BondedTokens.RoundInt64())
```

Comment:
Where is this `-5` coming from?

Reply:
From the additional double-sign slash I added to the test. Cleaned it up a bit.